### DISCUSSION TITLE
I am looking for flash usdt software

### DISCUSSION THREAD

**Discussion by Jawaan666** [community/community#82933] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I need software for flash usdt

**Comment by On-Fi** [comment, marked as answer]
Well...
you get help on fiverr:
https://www.fiverr.com/gigs/usdt-flashing
or here are a few repos:
https://github.com/topics/flash-usdt-trc20

**Reply by Jawaan666** [reply]
What's repo ?

**Reply by On-Fi** [reply]
A Repo is short for Repository, and that is a type of centrally located storage where you can keep all your project's files and resources.
You already made a Repo [Jawaan666](https://github.com/Jawaan666/Jawaan666)

This link down below explains in more detail what Repositories are. :) <br> https://docs.github.com/en/repositories/creating-and-managing-repositories/about-repositories

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @Jawaan666! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Reply by Jawaan666** [reply]
I am extremely new to all this
Would need guidance like a child

**Comment by freelanceworker95-boop** [comment]
> ### Select Topic Area
> Question
> 
> ### Body
> I need software for flash usdt

Join and Learn more un details about Flash USDT.
https://youtube.com/@cryptoclublive
